templates/todo/projects/csharp-cosmos-sql-func/infra/resources.bicep

```diff
@@ -0,0 +1,307 @@
+param location string
+param principalId string = ''
+param resourceToken string
+param tags object
+
+resource web 'Microsoft.Web/staticSites@2021-03-01' = {
+  name: 'stapp-${resourceToken}'
+  location: location
+  tags: union(tags, {
+      'azd-service-name': 'web'
+    })
+  sku: {
+    name: 'Free'
+    tier: 'Free'
+  }
+  properties: {
+    provider: 'Custom'
+  }
+}
+
+resource api 'Microsoft.Web/sites@2021-03-01' = {
+  name: 'app-api-${resourceToken}'
+  location: location
+  tags: union(tags, {
+      'azd-service-name': 'api'
+    })
```

Comment:
In latest bicep we move to single line for a bunch of these

Reply:
Resolved in latest commit.